apps/glusterfs/health.go

```diff
@@ -0,0 +1,123 @@
+package glusterfs
```

Comment:
This file looks to be missing the normal preamble we have in the source files in heketi.

Reply:
add license  path